modules/_labs/core/react/lib/theming/README.md

```diff
@@ -142,16 +145,31 @@ const theme: PartialCanvasTheme = {
 </CanvasProvider>;
 ```
 
+Example changing the direction:
+
+```tsx
+import {
+  CanvasProvider,
+  createCanvasTheme,
+  ContentDirection,
+} from '@workday/canvas-kit-labs-react-core';
+
+<CanvasProvider theme={createCanvasTheme({direction: ContentDirection.RTL})}>
+  {/* Your app with Canvas components */}
+</CanvasProvider>;
+```
+
 ### Nesting Theme Providers
```

Comment:
Can you provide an example here for reversing the top level content direction?

Reply:
I've done that below, in `### Nesting CanvasProvider components`